<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>287</width>
                <height>312</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Pick a pen</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0">
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
            <item row="1" column="0">
                <widget class="QWidget" name="widget" native="true">
                    <layout class="QGridLayout" name="gridLayout_2">
                        <item row="0" column="1">
                            <widget class="QSpinBox" name="spinBox_penWidth"/>
                        </item>
                        <item row="4" column="0" colspan="2">
                            <widget class="QGroupBox" name="groupBox">
                                <property name="title">
                                    <string>Sample</string>
                                </property>
                                <layout class="QGridLayout" name="gridLayout_3">
                                    <item row="0" column="0">
                                        <widget class="QWidget" name="sample_widget" native="true"/>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item row="0" column="0">
                            <widget class="QLabel" name="label_4">
                                <property name="text">
                                    <string>Pen Width</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>Pen Style</string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="0">
                            <widget class="QLabel" name="label_3">
                                <property name="text">
                                    <string>Pen Join</string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="1">
                            <widget class="QComboBox" name="comboBox_penJoin"/>
                        </item>
                        <item row="2" column="0">
                            <widget class="QLabel" name="label_2">
                                <property name="text">
                                    <string>Pen Cap</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QComboBox" name="comboBox_penStyle"/>
                        </item>
                        <item row="2" column="1">
                            <widget class="QComboBox" name="comboBox_penCap"/>
                        </item>
                    </layout>
                </widget>
            </item>
            <item row="1" column="1">
                <widget class="QWidget" name="widget_penColor" native="true"/>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>Dialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>143</x>
                    <y>291</y>
                </hint>
                <hint type="destinationlabel">
                    <x>143</x>
                    <y>155</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
